exasol_udf_mock_python/mock_meta_data.py

```diff
@@ -34,7 +34,7 @@ def __init__(
         assert output_type.upper() in ["EMITS", "RETURNS"]
         if is_variadic_input:
             for i in range(len(input_columns)):
-                assert str(i+1) == str(input_columns[i].name)
+                assert str(i) == str(input_columns[i].name)
```

Comment:
Alternatively you could write something like that:
```python
if is_variadic_input:
  assert all([ str(i) == str(c.name)) for i, c in enumerate(input_columns) ])
```

Reply:
good point, will adapt it